### DISCUSSION TITLE
Way to generate a sidebar?

### DISCUSSION THREAD

**Discussion by WoosterInitiative** [carltongibson/django-filter#1530] [Q&A]
I'm trying to more-or-less replicate the capabilities of the ModelAdmin List Filters outside of admin. Am I missing a way to generate the sidebar with context from django-filter? I think I've more or less figured out how to use it to do the filtering (I think), but not how to generate the list of clickable links.

Thanks.

**Comment by carltongibson** [comment]
Hi @WoosterInitiative — Did you see `LinkWidget`? https://django-filter.readthedocs.io/en/stable/ref/widgets.html#linkwidget

**Reply by WoosterInitiative** [reply]
I'll have to try that out, if I remember :( I'm not in the thick of that right now (made my own [package](https://github.com/wsutc/django-listview-filters) that does an OK job, so, until I'm ready to work on it again...). Before I dig too deeply into polishing that package, I'll check this out, though.

Thanks for the direction!
